ecs.php

```diff
@@ -40,6 +40,9 @@
     // make ECS 16x faster
     $parameters->set(Option::PARALLEL, true);
 
+    // Reusable ECS cache for Docker runtime
+    $parameters->set(Option::CACHE_DIRECTORY, 'tmp/ecs');
+
```

Comment:
Could you restore the defaults? This now conflicts with-non-docker usage

Reply:
@TomasVotruba with Docker cache only makes sense with explicitly specified path, because it needs to be mounted as volume. Otherwise every run will start from scratch... And it's not safe to specify path outside of the project (absolute path) because you can't be sure which path is always available.

Reply:
Moved to #1928